### DISCUSSION TITLE
Kontaktinformasjon og hjelp

### DISCUSSION THREAD

**Discussion by Febakke** [digdir/designsystemet#1677] [Patterns for public services]
### Relevant innsikt
- https://github.com/digdir/innsiktsbibliotek/issues/34

https://github.com/digdir/designsystemet/issues/1239

**Comment by mrosvik** [comment]
## Workshop om kontaktinfo 17.06.25 
Vi har hatt en arbeids-workshop i dag (17.06.25) i produktgruppen "Portaler og brukeropplevelser" med fokus på kontaktinformasjon. Vi diskuterte flere problemstillinger som etter hvert bør danne grunnlag for retningslinjer og eksempler i en mønsterartikkel. 

### Bakgrunnen for problemet fra Team Arbeidsflate:
«Det er så mange som ikke forstår at vi bare er postkassen»​. 1 av 3 brukere må rutes videre​ når de tar kontakt med kundeservice. De kommer til feil kontaktpunkt og blir av og til kasteball mellom etater. Kan vi løse noe av dette i meldingene/Innboksen?

Kan vi lage noe som kan gjenbrukes flere steder som henter informasjonen ett sted?​
- Hvilke behov er det til informasjonen, hvilke data trenger vi?​
- Hvor skal informasjonen ligge?​
- Hvordan sikrer vi riktig og oppdaterte data?​
- Kan vi lage noe sammen som gjør at vi oppdaterer kun ett sted?​

### Oppsummering av temaene som ble diskutert under "utforming"

- **Veiledningspikt og brukerbehov**
Iht. til forvaltningsloven § 11 om veiledningsplikt, diskuterte vi hvordan brukere har ulike behov og preferanser for kontaktpunkter – noen vil ringe, andre vil skrive. Det er viktig å gi valget tidlig: Vil du skrive eller snakke? Ønsker du hjelp av en AI eller et menneske?

- **Chat og forventningsstyring**
   - Hvordan gjør vi det tydelig hva slags chat det er snakk om? AI eller menneske?
   - Er det mulig å komme i kontakt med et menneske via chatten?
   - Vi bør anbefale at brukeren ikke må "gjennom" AI for å nå et menneske.
   - Vi diskuterte også språkbruk: "Vil du snakke med et menneske?" (ikke bruk _ekte_ menneske)

- **Plassering av kontaktinfo**
Kontaktinformasjon bør plasseres i kontekst – der brukeren faktisk får behov for hjelp (f.eks. i skjema, ved feilmeldinger eller ved veiledning). Bør det også være en generell kontaktinformasjon tilgjengelig, og i så fall hvor? Footer? Vi diskuterte at i stedet for telefonnummer i footeren kan det i noen tilfeller være bedre med en lenke til informasjon om hjelp, slik at hjelpen kontekstualiseres.

- **Unngå å "gjemme" kontaktinfo**
Eksempel fra Brønnøysundregistrene viste at å skjule kontaktinfo bare gir merarbeid både for brukerne, og ikke mindre arbeid for kundeservice. De finner kontaktinfoen til slutt, det tar dem bare mer tid...

    - Anbefaling: Ikke tving brukeren til å lete, og unngå bruk av interne begreper.

- **Kontaktinfo ved systemfeil og vedlikehold**
Må vi alltid vise telefonnummeret til kundesenteret ved systemfeil eller vedlikehold? Hva gjør vi dersom feilen oppstår utenom kundesenterets åpningstider? Det vi vet bør vi allerede skrive i feilmeldingene/systemvarslene, ... kundeservice vet ikke noe mer, og det er kanskje derfor unødvendig å oppfordre til å ringe? Med mindre vi vet at ved å ringe får du faktisk hjelp.

- **Eksempler**
Vi trenger eksempler på god praksis. 
   - Vis om det faktisk finnes mennesker bak tjenesten 
   - Åpningstider: vis alltid når man kan forvente svar.
   - Kontaktlenker må være tydelig merket og beskrivende for skjermlesere og lenkelister

- **Ikoner** 
Ikonbruk kan bidra til å vise hvem brukeren snakker med – f.eks. menneskechat vs. AI-chat. Enhetlig ikonbruk på tvers av etater vil styrke mønsteret.

- **Når brukeren kontakter kundeservice**
Hvordan kan vi sikre at den som skal hjelpe vet så mye som mulig om hvor brukeren står fast, og situasjonen de står i, så brukeren slipper å forklare veldig mye. F.eks at den personen som svarer kan si:  "Hei, jeg ser du sitter med aksjonæroppgaven, hva lurer du på?"

## Neste steg
Første steg blir å utvikle en komponent i Altinn Component Library, som skal testes i Portal og i Arbeidsflate. Erfaringene herfra vil gi verdifull innsikt som vi kan bruke videre i arbeidet med et mønster for kontaktinformasjon som kan favne bredere og dekke flere behov på tvers av offentlig sektor.

Eksempel fra Altinn der det er Skatteetaten som er relevant som kontaktpunkt: 
<img width="1173" alt="image" src="https://github.com/user-attachments/assets/577363f2-0d53-42c7-9a2f-02e31382b651" />
